shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -0,0 +1,242 @@
+          .desc("The level of the log what you specified.").build();
+  private static final Options OPTIONS = new Options()
+      .addOption(TATGET_OPTION)
+      .addOption(LOG_NAME_OPTION)
+      .addOption(LEVEL_OPTION);
+
+  private static final String USAGE =
+      "logLevel --logName=LOGNAME [--target=<master|workers|host:port>] [--level=LEVEL]";
+  public static final String TARGET_SEPARATOR = ",";
+
+  /**
+   * Prints the help message.
+   *
+   * @param message message before standard usage information
+   */
+  public static void printHelp(String message) {
+    System.err.println(message);
+    HelpFormatter help = new HelpFormatter();
+    help.printHelp(LOG_LEVEL, OPTIONS, true);
+  }
+
+  /**
+   * Implements log level setting and getting.
+   *
+   * @param args list of arguments contains target, logName and level
+   * @return 0 on success, 1 on failures
+   */
+  public static int logLevel(String... args) {
+    CommandLineParser parser = new DefaultParser();
+    CommandLine cmd;
+
+    try {
+      cmd = parser.parse(OPTIONS, args, true /* stopAtNonOption */);
+    } catch (ParseException e) {
+      printHelp("Unable to parse input args: " + e.getMessage());
+      return 1;
+    }
+    List<TargetInfo> targetInfoList = new ArrayList<>();
+
+    String[] targets = parseOptTarget(cmd);
+    String logName = parseOptLogName(cmd);
+    String level = parseOptLevel(cmd);
+
+    for (String target : targets) {
+      if (target.contains("master")) {
+        String masterHost = NetworkAddressUtils.getConnectHost(ServiceType.MASTER_WEB);
+        int masterPort = NetworkAddressUtils.getPort(ServiceType.MASTER_WEB);
+        targetInfoList.add(new TargetInfo(masterHost, masterPort, "master"));
+      } else if (target.contains("workers")) {
+        AlluxioBlockStore alluxioBlockStore = AlluxioBlockStore.create();
+        try {
+          List<BlockWorkerInfo> workerInfoList = alluxioBlockStore.getWorkerInfoList();
+          for (BlockWorkerInfo workerInfo : workerInfoList) {
+            WorkerNetAddress netAddress = workerInfo.getNetAddress();
+            targetInfoList.add(
+                new TargetInfo(netAddress.getHost(), netAddress.getWebPort(), "worker"));
+          }
+        } catch (IOException e) {
+          e.printStackTrace();
+          return 1;
+        }
+      } else if (target.contains(":")) {
+        String[] hostPortPair = target.split(":");
+        int port = Integer.parseInt(hostPortPair[1]);
+        targetInfoList.add(new TargetInfo(hostPortPair[0], port, "worker"));
+      }
+    }
+
+    for (TargetInfo targetInfo : targetInfoList) {
+      if (handleTarget(logName, level, targetInfo)) {
+        return 1;
+      }
+    }
+    return 0;
+  }
+
+  private static String[] parseOptTarget(CommandLine cmd) {
+    String[] targets;
+    if (cmd.hasOption(TARGET_OPTION_NAME)) {
+      String argTarget = cmd.getOptionValue(TARGET_OPTION_NAME);
+      if (StringUtils.isBlank(argTarget)) {
```

Comment:
since `hasArg(true)`, so if `--target` exists, `argTarget` must not be blank, right?